roaster-oracle/pom.xml

```diff
@@ -28,6 +29,11 @@
 			<artifactId>roaster-api</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.eclipse.microprofile.config</groupId>
+			<artifactId>microprofile-config-api</artifactId>
```

Comment:
ide nem inkább roaster-common-config kellene inkább?

Reply:
a roaster-oracle-nek csak a org.eclipse.microprofile.config.Config a függősége, ami a microprofile-config-api-ban található, nem tennék be implementációs (roaster-common-config) függőséget